services/dy-jupyter/devel/requirements.txt

```diff
@@ -4,7 +4,7 @@ autopep8>=1.3.5
 certifi
 coveralls~=1.3
 docker~=3.7
-minio==4.0.0
+minio==4.0.16
```

Comment:
should we set the dependabot in this repo as well to keep up with the updates?

Reply:
very good idea